helix-term/src/ui/picker.rs

```diff
@@ -204,6 +219,14 @@ impl<T: Item> FilePicker<T> {
                 let doc = editor.documents.get(&id).unwrap();
                 Preview::EditorDocument(doc)
             }
+            PathOrId::Text(rope) => {
+                let doc = Document::from(
+                    rope,
+                    None,
+                    Arc::new(ArcSwap::new(Arc::new(Config::default()))),
```

Comment:
I am not sure if this is the correct value for this argument.

Reply:
This should be `editor.config.clone()` instead